File: BharatEpaisaApp/Database/Models/AnonymousWallet.cs
namespace BharatEpaisaApp.Database.Models
{
    internal class AnonymousWallet
    {
        public string uniqueId { get; set; }
        public string cloudMsgToken { get; set; }
        public string publicKey { get; set; }
        public bool isActive { get; set; }

        public AnonymousWallet(string idValue, string token, string key, bool isAValue)
        {
            uniqueId = idValue;
            cloudMsgToken = token;
            publicKey = key;
            isActive = isAValue;
        }
    }
}

File: Server/functions/models/anonymous_wallet.js
const Joi = require("joi");

const Anonymous_Wallet = Joi.object({
	uniqueId: Joi.string(),
	cloudMsgToken: Joi.string().required(),
	publicKey: Joi.string().required(),
	isActive: Joi.bool().required(),
});

module.exports = Anonymous_Wallet;
